.github/ISSUE_TEMPLATE/bug_report.yml

```
@@ -0,0 +1,87 @@
+# SPDX-FileCopyrightText: Copyright (c) 2023 NVIDIA CORPORATION & AFFILIATES. All rights reserved.
+# SPDX-License-Identifier: Apache-2.0
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+name: Bug Report
+description: File a bug report for cuDF
+title: "[BUG]: "
+labels: ["bug"]
+
+body:
+  - type: markdown
+    attributes:
+      value: |
+        Thanks for taking the time to help cuDF and fill out this bug report!
+        - By submitting this issue, you agree to follow our [Code of Conduct](https://docs.rapids.ai/resources/conduct/)
+        - You also confirm that you have searched the [open bugs](https://github.com/rapidsai/cuDF/issues?q=is%3Aopen+is%3Aissue+label%3Abug) and have found no duplicates for this request
+
+  - type: input
+    id: version
+    attributes:
+      label: Version
+      description: What version of cuDF are you running?
+      placeholder: "example: 23.06"
+    validations:
+      required: true
+
+  - type: dropdown
+    id: installation-method
+    attributes:
+      label: On which installation method(s) does this occur?
+      multiple: true
+      options:
+        - Docker
+        - Conda
+        - Source
+        - Rapids-Compose
+
+  - type: textarea
+    id: description
+    attributes:
+      label: Describe the issue
+      description: Please provide a complete and succinct description of the problem, including what you expected to happen.
+      placeholder: XYZ occurred, I expected QRS results.
+    validations:
+      required: true
+
+  - type: textarea
+    id: mvr
+    attributes:
+      label: Minimum reproducible example
+      description: Please supply a [minimum reproducible code example](https://matthewrocklin.com/blog/work/2018/02/28/minimal-bug-reports) here.
+      render: shell
+
+  - type: textarea
+    id: logs
+    attributes:
+      label: Relevant log output
+      description: Please paste relevant error and log output here
+      render: shell
+
+  - type: textarea
+    id: env-details
+    attributes:
+      label: Environment details
+      description: Please specify installation info and paste the results from [print_env.sh](https://github.com/rapidsai/cuDF/blob/main/print_env.sh) here
+      placeholder: |
+       + Environment location: [Bare-metal, Docker, Cloud(specify cloud provider)]
+       + If method of install is Docker, provide `docker pull` & `docker run` commands used
+       + Paste in print_env.sh (linked above) output here
```

Tightening up the prompt here.
```suggestion
       + Paste output from `print_env.sh` (linked above)
```